src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Google LLC
+# Copyright 2021 Google LLC
```

Comment:
We don't change Copyright years when editing files.

Reply:
ok